src/main/java/frc/robot/shooter/EjectCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class EjectCommand extends Command {
+    private final ShooterSubsystem subsystem;
+    private final double targetSpeed;
+
+    public EjectCommand(ShooterSubsystem subsystem, double targetSpeed) {
```

Comment:
rename this class `ShooterEject` for clarity vs the undertaker

Reply:
renamed